ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusStepsScreenView.swift

```diff
@@ -0,0 +1,270 @@
+            cancelButton
+              .frame(width: 32, height: 32)
+              .padding(24)
+          }
+
+          FocusStepsPagingIndicator(totalPages: 4, activeIndex: .constant(2))
+        }
+        .frame(maxWidth: .infinity, maxHeight: .infinity)
+        .background {
+          Image("focus-background-large", bundle: .module)
+            .resizable()
+            .aspectRatio(contentMode: .fill)
+            .opacity(0.20)
+        }
+        .background {
+          NavigationLink("", isActive: $isP3AViewPresented) {
+            FocusP3AScreenView(
+              attributionManager: attributionManager,
+              p3aUtilities: p3aUtilities,
+              shouldDismiss: $shouldDismiss
+            )
+          }
+        }
+      } else {
+        VStack {
+          stepsContentView
+          FocusStepsPagingIndicator(totalPages: 4, activeIndex: $indicatorIndex)
+            .opacity(opacity)
+            .onAppear {
+              withAnimation(.easeInOut(duration: 1.5).delay(1.25)) {
+                opacity = 1.0
+              }
+            }
+        }
+        .padding(.bottom, 20)
+        .background(Color(braveSystemName: .pageBackground))
+        .background {
+          NavigationLink("", isActive: $isP3AViewPresented) {
+            FocusP3AScreenView(
+              attributionManager: attributionManager,
+              p3aUtilities: p3aUtilities,
+              shouldDismiss: $shouldDismiss
+            )
+          }
+        }
+        .overlay(alignment: .topTrailing) {
+          cancelButton
+            .frame(width: 32, height: 32)
+            .padding(24)
+        }
+      }
+    }
+    .navigationViewStyle(StackNavigationViewStyle())
+    .navigationBarHidden(true)
+  }
+
+  private var stepsContentView: some View {
+    VStack {
+      if shouldUseExtendedDesign {
+        HStack {
+          Image("focus-icon-brave", bundle: .module)
+            .resizable()
+            .matchedGeometryEffect(id: "icon", in: namespace)
+            .frame(width: 78, height: 78)
+
+          Image("focus-brave-watermark", bundle: .module)
+            .resizable()
+            .frame(width: 111, height: 31)
+        }
+      } else {
+        Image("focus-icon-brave", bundle: .module)
+          .resizable()
+          .matchedGeometryEffect(id: "icon", in: namespace)
+          .frame(width: 78, height: 78)
+      }
+      VStack(spacing: 0) {
+        FocusStepsHeaderTitleView(activeIndex: $indicatorIndex)
+          .padding(.bottom, 24)
+
+        TabView(selection: $indicatorIndex) {
+          FocusAdTrackerSliderContentView()
+            .tag(0)
+          FocusVideoAdSliderContentView()
+            .tag(1)
+        }
+        .background(Color(braveSystemName: .pageBackground))
+        .clipShape(RoundedRectangle(cornerRadius: 16.0))
+        .tabViewStyle(PageTabViewStyle(indexDisplayMode: .never))
+        .animation(.easeInOut, value: indicatorIndex)
+        .transition(.slide)
```

Comment:
This transition is redundant and wont be used as `stepsContentView` is not dynamically added/removed